bitbucket-slack-server-integration-plugin/src/main/resources/com/atlassian/bitbucket/plugins/slack/bitbucket-slack.properties

```diff
@@ -249,6 +251,7 @@ slack.activity.pr.reviewed.long=:warning: {0} *reviewed* pull request {1} {3} \u
 slack.activity.pr.unapproved.needswork.long={0} *removed needs work review* from pull request {1} {3} \u2192 {4} in {2}
 slack.activity.pr.merged.long=:tada: {0} *merged* pull request {1} {3} \u2192 {4} in {2}
 slack.activity.pr.updated.long={0} *updated* pull request {1} {3} \u2192 {4} in {2}
+slack.activity.pr.rescoped.long={0} *updated* pull request {1} {3} \u2192 {4} in {2}
```

Comment:
There is already a generic "updated" notification. In this case a more precise wording may be used to prevent misunderstanding. Let's use explicit "rescoped" wording here and in the related short message pattern.